composer.json

```diff
@@ -9,13 +9,13 @@
         "cakephp/cakephp": "3.6.*",
         "cakephp/migrations": "^2.0.0",
         "cakephp/plugin-installer": "^1.0",
-        "josegonzalez/dotenv": "3.*",
         "mobiledetect/mobiledetectlib": "2.*"
     },
     "require-dev": {
         "cakephp/bake": "^1.1",
         "cakephp/cakephp-codesniffer": "^3.0",
         "cakephp/debug_kit": "^3.15.0",
+        "josegonzalez/dotenv": "3.*",
```

Comment:
Nice moving this to `require-dev`.

Reply:
Wouldnt it then also have to be in suggest for clarification?

Reply:
OK, I guess this is fine :)